.gitlab-ci.yml

```diff
@@ -0,0 +1,85 @@
+stages:
+  - analyze
```

Comment:
What's the purpose of this CI?

Reply:
this is for deploying the project on the k8s cluster, running all related scripts in this ci job. i didn't use this. i  deploy manually which is easy for debugging each step.